components/SensorComponent.js

```diff
@@ -1,96 +1,92 @@
 import React, { useState, useEffect } from "react";
 import { Accelerometer } from "expo-sensors";
 import styled from "styled-components/native";
+import { View, Text, TouchableOpacity } from "react-native";
 
-// ==========================
-// = Functions
 const isShaking = (data) => {
-  // x,y,z CAN be negative, force is directional
-  // We take the absolute value and add them together
-  // This gives us the total combined force on the device
   const totalForce = Math.abs(data.x) + Math.abs(data.y) + Math.abs(data.z);
-
-  // If this force exceeds some threshold, return true, otherwise false
-  // Increase this threshold if you need your user to shake harder
   return totalForce > 1.78;
 };
 
-// ==========================
-// = Styled components
-const ShakeView = styled.View`
+const Container = styled.View`
   display: flex;
-  flex-direction: column;
+  text-align: center;
+  margin-bottom: 20px;
+  overflow-wrap:break-word;
+  margin-left: 20px;
+  margin-right:20px;
 `;
 
-const ShakeAlert = styled.Text`
-  font-size: 36px;
-  font-weight: bold;
-  color: #aa0000;
-`;
-const ShakeDataView = styled.View``;
-const ShakeDataTitle = styled.Text`
-  font-weight: bold;
-`;
-const ShakeData = styled.Text``;
+
+const Button = styled.TouchableOpacity`
+display: flex;
+text-align: center;
+border: solid 2px; 
+width: 50%;
+border-radius:5px;
+padding:8px;
+margin-top: 60px;
+margin-left: 80px;
+box-shadow: 5px 5px #000;
+background-color: grey;
+`
+  ;
 
 export const SensorComponent = () => {
-  // This function determines how often our program reads the accelerometer data in milliseconds
-  // https://docs.expo.io/versions/latest/sdk/accelerometer/#accelerometersetupdateintervalintervalms
+  const [quote, setQuote] = useState({});
```

Comment:
It's good practice to keep all of your state together at the top of your component, so instead of having three spread out through your code, group them here, like so: 
```
  const [quote, setQuote] = useState({});
  const [subscription, setSubscription] = useState(null);
  const [data, setData] = useState({
    x: 0,
    y: 0,
    z: 0,
  });